cmd/kubelet/app/init_windows.go

```diff
@@ -19,14 +19,44 @@ limitations under the License.
 package app
 
 import (
+	"fmt"
+
+	"golang.org/x/sys/windows"
+	"k8s.io/klog/v2"
 	"k8s.io/kubernetes/pkg/windows/service"
 )
 
 const (
 	serviceName = "kubelet"
 )
 
-func initForOS(windowsService bool) error {
+// getPriorityValue returns the value associated with a Windows process priorityClass
+// Ref: https://docs.microsoft.com/en-us/windows/win32/cimwin32prov/setpriority-method-in-class-win32-process
+func getPriorityValue(priorityClassName string) uint32 {
+	var priorityClassMap = map[string]uint32{
+		"IDLE_PRIORITY_CLASS":         uint32(64),
+		"BELOW_NORMAL_PRIORITY_CLASS": uint32(16384),
+		"NORMAL_PRIORITY_CLASS":       uint32(32),
```

Comment:
if i understand correctly, the preference is to have normal/above/high, but its not clear that there is an actual use case for below normal or idle or real time.  how are priority classes used by the containers themselves on windows?  are users able to define an associated priority or is it inferred?

Reply:
>  how are priority classes used by the containers themselves on windows? are users able to define an associated priority or is it inferred?

Nope, it is always the default which is `Normal`